app/policies/organizations/activations_policy.rb

```diff
@@ -0,0 +1,18 @@
+module Organizations
+  class ActivationsPolicy < ApplicationPolicy
+    pre_check :verify_organization!
+    pre_check :verify_active_staff!
+
+    def update_activation?
+      return false if record.id == user.id
+
+      record_role = record.roles.first.name
+
+      if record_role == "super_admin" || record_role == "admin"
```

Comment:
you could shorten this a little to 
`if %w[super_admin admin].include?(record_role)`